src/data/terraform.json

```diff
@@ -106,7 +106,8 @@
 			"name": "Configuration Language",
 			"path": "language",
 			"visuallyHideSidebarTitle": true,
-			"addOverviewItem": false
+			"addOverviewItem": false,
+			"description": "This a custom description of the Terraform Language page that is written in src/data/terraform.json. If this weren't here, we'd fall back to the frontmatter description."
```

Comment:
Added in this custom description as a demo. Expectation is that we'll sort out this content at the same time we address other Terraform placeholder content.